docs/contract_types.rst

```diff
@@ -2,6 +2,22 @@
 Contract types
 ==============
 
+Shared options
+--------------
+
+The following options can be used in all contracts.
+
+- ``ignore_imports``: Optional list of imports, each in the form ``mypackage.foo.importer -> mypackage.bar.imported``.
+  These imports will be ignored: if the import would cause a contract to be broken, adding it to the list will cause the
+  contract be kept instead.
+
+  Wildcards are supported as well.
```

Comment:
The terminology seems fine to me, let me add it. The double asterisk seems to be nice to have for later.
Personally I would stick to regex for now, since it's very easy. Even with a double asterisk it should be possible

Reply:
> So you want to drop "foo.bar*"?

Yes, I think so. My view is we should be wary of introducing it as a feature if no-one actually needs it. Also, if we were to support this then it would make it possible for people to accidentally write `foo.bar*` instead of `foo.bar.*`, which would have a different meaning - better to report that it's an invalid expression. We can always add it later if it turns out that there's demand for it.
  
Open to your thoughts on this too. In your case, would you actually use it or would the module-level asterisks meet your needs?

Reply:
I agree, that makes sense

Reply:
done